frontend/src/reducer.js

```diff
@@ -0,0 +1,11 @@
+const reducer = (state, action) => {
+  if (action.type === "login") {
+    return { ...state, loggedIn: true };
+  }
+  if (action.type === "logout") {
+    return { ...state, loggedIn: false };
+  }
+  return state;
+};
+
+export default reducer;
```

Comment:
You'll definitely want additional reducers as your app grows in complexity. Not a big deal for now, but you can set yourself up well by giving this a name (like `authReducer`). You can use the `combineReducers` helper from Redux to merge multiple reducers. Lemme know if you want more info about this (but truthfully it's not the highest priority, not yet)